libs/designer-ui/src/lib/dictionary/expandeddictionary.tsx

```diff
@@ -75,36 +87,22 @@ export const ExpandedDictionary = ({
                 getTokenPicker={getTokenPicker}
                 onFocus={() => addItem(index)}
                 valueType={keyType}
-              >
-                <SerializeExpandedDictionary
-                  items={items}
-                  initialItem={item.key}
-                  index={index}
-                  type={ExpandedDictionaryEditorType.KEY}
-                  setItems={setItems}
-                />
-              </BaseEditor>
+                editorBlur={(newState: ChangeState) => handleBlur(newState, index, ExpandedDictionaryEditorType.KEY)}
+              ></StringEditor>
```

Comment:
Super tiny nit, can we make these close with `/>` instead of a separate closing tag